frontend/src/modules/organization/components/edit/organization-add.vue

```diff
@@ -0,0 +1,261 @@
+  modelValue: boolean,
+}>();
+
+const emit = defineEmits<{(e: 'update:modelValue', value: boolean): void }>();
+
+const isModalOpen = computed({
+  get: () => props.modelValue,
+  set: (value) => emit('update:modelValue', value),
+});
+
+const router = useRouter();
+
+const { selectedProjectGroup } = storeToRefs(useLfSegmentsStore());
+
+type OrganizationAddIdentity = OrganizationIdentity & {
+  image: string,
+  prefix: string,
+  placeholder: string,
+}
+
+const platformList: OrganizationAddIdentity[] = Object.entries(CrowdIntegrations.organizationIdentities)
+  .map(([key, config]: [string, any]) => ({
+    prefix: config.orgUrlPrefix ?? config.urlPrefix,
+    placeholder: config.placeholder,
+    image: config.image,
+    platform: key as Platform,
+    type: OrganizationIdentityType.USERNAME,
+    value: '',
+    verified: true,
+    sourceId: null,
+  }));
+
+interface OrganizationAddForm {
+  subproject: string;
+  name: string;
+  website: string;
+  identities: OrganizationAddIdentity[];
+}
+
+const form = reactive<OrganizationAddForm>({
+  subproject: '',
+  name: '',
+  website: '',
+  identities: platformList,
+});
+
+const rules = {
+  subproject: {
+    required,
+  },
+  name: {
+    required,
+  },
+  website: {
+    required,
+  },
+  identities: {
+    required: (value: OrganizationAddIdentity[]) => value.some((i) => i.value.length),
+  },
+};
+
+const $v = useVuelidate(rules, form);
+
+const resetForm = () => {
+  form.subproject = '';
+  form.name = '';
+  form.website = '';
+  form.identities = platformList;
+  $v.value.$reset();
+};
+
+const sending = ref<boolean>(false);
+const createOrganization = () => {
+  if ($v.value.$invalid) {
+    return;
+  }
+
+  const data: Partial<Organization> = {
+    manuallyCreated: true,
+    attributes: {
+      name: {
+        default: form.name,
+        custom: [form.name],
+      },
+    },
+    displayName: form.name,
+    identities: [
+      ...form.identities
+        .filter((i) => !!i.value)
+        .map((i) => ({
+          type: OrganizationIdentityType.USERNAME,
+          platform: i.platform as Platform,
+          value: i.value,
+          verified: true,
+        })),
+      {
+        type: OrganizationIdentityType.PRIMARY_DOMAIN,
+        platform: Platform.CUSTOM,
+        value: form.website,
+        verified: true,
+      },
+    ],
+    segments: [form.subproject],
+  };
+
+  sending.value = true;
+
+  OrganizationApiService.create(data, [form.subproject])
+    .then((newOrg: Organization) => {
+      router.push({
+        name: 'organizationView',
+        params: {
+          id: newOrg.id,
+        },
+        query: {
+          projectGroup: selectedProjectGroup.value?.id,
+        },
+      });
+      isModalOpen.value = false;
+      resetForm();
+    })
+    .catch((error: any) => {
+      if (error.response.status === 409) {
+        Message.error('Organization was not created because the identity already exists in another profile');
```

Comment:
Can you update this and the `contributor-identity-add`, so that below the toast message we have a button to View the profile? Backend returns the id right?

Reply:
Actually I just tried and I believe they already return the existing organization in the payload. So this error message won't even exist.
Do you think there's a way to distinguish if the organization already existed or not? Because that way we could show an informative message mentioning that an organization with the same identity already existed